internal/helm/controller/reconcile.go

```diff
@@ -56,7 +56,10 @@ type HelmOperatorReconciler struct {
 }
 
 const (
-	finalizer = "uninstall-helm-release"
+	// uninstallFinalizer is added to CRs so they are cleaned up after uninstalling a release.
+	uninstallFinalizer = "helm.sdk.operatorframework.io/uninstall-release"
+	// Deprecated: use uninstallFinalizer. This will be removed in operator-sdk v2.0.0.
+	uninstallFinalizerLegacy = "uninstall-helm-release"
```

Comment:
This finalizer, if on any existing CRs, will still be respected until v2.0.0 (or helm-operator is broken out into a separate repo and another major version released).